cli/compose/template/template_test.go

```diff
@@ -229,6 +237,7 @@ func TestExtractVariables(t *testing.T) {
 			},
 		},
 		{
+			name: "default-variable2",
```

Comment:
nit: I'm sure you can find a better name 😛

Reply:
I'll let you that pleasure :stuck_out_tongue:

Reply:
I can't, I'm only a humble reviewer 😇